WordPress/src/main/java/org/wordpress/android/ui/pages/PageListFragment.kt

```diff
@@ -129,7 +142,58 @@ class PageListFragment : ViewPagerFragment() {
         adapter.update(pages)
     }
 
+    fun onItemTapped(pageItem: Page) {
```

Comment:
It would be nice if we could move this logic in the `PageListViewModel` to handle the `snackbar` visibility from there.